app/assets/stylesheets/psique/foundation/_config.scss

```diff
@@ -9,25 +9,68 @@
 //   Colors
 // -------------------------------------
 
-// ----- Palette ----- //
+// ----- Palette Vars ----- //
+
+// Main Colors
```

Comment:
Although I'm not a comments lover, this seem a good place to add a link (commented, of course) pointing to where we can see the colors palette